src/components/06-miscellaneous/03-toolbar/toolbar.njk

```diff
@@ -1,256 +1,135 @@
-<div class="grid-row margin-105">
```

Comment:
Render existing components

Reply:
the toolbar needs some work

Reply:
a separate story is required to address this